cvat/apps/events/event.py

```diff
@@ -47,25 +47,27 @@ def record_server_event(
     *,
     scope: str,
     request_id: Optional[str],
+    user_agent: Optional[str] = None,
```

Comment:
Can you merge these into a single parameter (taking a compound object)? Seems like it would be more robust to pass request information as a single parameter, instead of deciding for every call whether the user agent is needed or not.